template.yaml

```diff
@@ -80,7 +80,7 @@ Resources:
     Properties:
       FunctionName: SungardAS-AWSConfig
       Handler: index.handler
-      Runtime: nodejs4.3
+      Runtime: nodejs8.10
```

Comment:
Make this a parameter to the template

Reply:
Runtime cannot be a Parameter. still if we need to try this , then we can do while we move code to BitBucket.

Reply:
We will take care of this when we move code to BitBucket.